docs/articles/get-started.rst

```diff
@@ -0,0 +1,433 @@
+Getting Started
+###############
+
+Pytest-mh is not a plugin to support unit testing. It is a plugin designed to
+support testing your application as a complete product, this is often referred
+to as a black-box, application or system testing. The application is installed
+on the target host and tested there by running commands on the host (and on
+other hosts that are required). These hosts can be virtual machines or
+containers.
+
+.. seealso::
+
+    SSH, podman and docker may be used to execute commands on the remote hosts. See
+    :doc:`running-commands` for more information.
+
+As such, it is often useful to write a high level API that will support testing
+your application and make your test smaller and more readable. This may require
+a non-trivial initial investment, but it will pay off in the long run. However,
+implementing such API is not required and it is perfectly possible to run
+commands on the host directly from each test. That, however, makes tests usually
+larger and more difficult to understand and maintain -- but every project is
+different and it is up to you to choose how are you going to test your
+application.
+
+.. seealso::
+
+    Pytest-mh provides several building blocks to help you design your test
+    framework. See :doc:`extending`. **It would be good to open this document
+    and read it side by side.**
+
+We will use `sudo <https://www.sudo.ws/>`__ as the application to test for this
+getting started guide as sudo is known by every power user so changes are that
+you are already familiar with it. Additionally, sudo tests allows us to show
+many pytest-mh features. Note, that these tests were written only as an example
+and sudo itself is not using pytest-mh for its tests at this moment and there
+are no plans to do so.
+
+.. seealso::
+
+    The example code can be found in the `example
+    <https://github.com/next-actions/pytest-mh/tree/master/example>`__ folder of
+    the git repository.
+
+Example project: sudo
+=====================
+
+`Sudo <https://www.sudo.ws/>`__ is a widely known tool that can elevate current
+user's privileges by running command as different user -- usually ``root``. It
+is possible to write a set of rules to define which user can run which command
+and these rules can be stored either locally in ``/etc/sudoers`` or in `LDAP
+<https://en.wikipedia.org/wiki/Lightweight_Directory_Access_Protocol>`__
+database.
+
+Our goals are:
+
+* write basic tests
+
+  * allow user run all commands, user must authenticate
+  * allow user run all commands, without authentication
+  * allow user run all commands if he or she is a member of a group, user must authenticate
```

Comment:
FYI, the generic pronoun in English is they

Reply:
I learnt it from an English teacher who is Czech, so you could say we're even :wink: